pwndbg/wrappers/__init__.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import pwndbg.which
+
+
+class OnlyWithCommand(object):
+    def __init__(self, func):
+        self.func = func
```

Comment:
@degrigis JFYI - about objects on which you can't set an attribute - there is a possibility to specify `__slots__` class static attribute which makes instances of this class not having a `__dict__` attribute.

For more, see [`__slots__` magic](http://book.pythontips.com/en/latest/__slots__magic.html) - a page I contributed/extended for Intermediate Python book.